UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.API.FunctionalTests/FunctionalTests/CallEssEndPoint.cs

```diff
@@ -6,6 +6,7 @@
 
 namespace UKHO.PeriodicOutputService.API.FunctionalTests.FunctionalTests
 {
+    [Ignore("temp")]
```

Comment:
This will be removed right ? Applicable wherever it added in other FTs. as well.

Reply:
added temporary and has been removed.